sample/cal/src/App.tsx

```diff
@@ -1,64 +1,78 @@
-import React, {
-  useEffect,
-  useState
-} from 'react';
+import React, {useEffect, useState} from 'react';
 import type {ReadTransaction, WriteTransaction} from 'replicache';
-import add from 'date-fns/add'
+import add from 'date-fns/add';
 import setDay from 'date-fns/setDay';
 import areIntervalsOverlapping from 'date-fns/areIntervalsOverlapping';
 import format from 'date-fns/format';
 import uuid from 'uuid-random';
 
 import './App.css';
-import type {
-  Event,
-  NormalizedEvent,
-  EventDate
-} from './types';
-import {
-  rep,
-  addEvent,
-  updateEvent,
-  deleteEvent,
-} from './data';
+import type {Event, NormalizedEvent, EventDate} from './types';
+import {rep, addEvent, updateEvent, deleteEvent} from './data';
 
 export default App;
 
 function App() {
-  const events = useSubscribe(async (tx: ReadTransaction) => {
-    // TODO: Scan API needs improvement.
-    // See https://github.com/rocicorp/replicache-sdk-js/issues/30.
-    const res = Array.from(await tx.scan({prefix: '/event/', limit: 50000}));
-    const result = res.map(item => {
-      let event = item.value as Event;
-      if (!event.start || !event.end) {
-        return null;
-      }
-      let normalized = item.value as unknown as NormalizedEvent;
-      normalized.start = eventDate(event.start);
-      normalized.end = eventDate(event.end);
-      return normalized;
-    }).filter(item => {
-      if (!item) {
-        return false;
+  const events = useSubscribe(
+    async (tx: ReadTransaction) => {
+      // TODO: Scan API needs improvement.
+      // See https://github.com/rocicorp/replicache-sdk-js/issues/30.
+      const result: NormalizedEvent[] = [];
+
+      for await (const value of tx.scan({prefix: '/event/'})) {
```

Comment:
It would be good to include the helper library that makes map work since this is a sample.